docs/snippets/AUTO0.cpp

```diff
@@ -0,0 +1,11 @@
+#include <ie_core.hpp>
+
+int main() {
+//! [part0]
+    InferenceEngine::Core ie;
+    InferenceEngine::CNNNetwork network = ie.ReadNetwork("sample.xml");
+    //NEW IE-CENTRIC API, the "auto" plugin is (globally) pre-configured with the explicit option:
```

Comment:
I see no global "pre-configuring" of the AUTO plugin in the example (presumably with ie.SetConfig(cong, "AUTO"))

Reply:
this example seems to be also used below